### DISCUSSION TITLE
Google Drive 分享打开文件夹是空的。

### DISCUSSION THREAD

**Discussion by HumanMus1c** [rnchg/APT#5] [General]
![image](https://github.com/rnchg/Apt/assets/46023154/0cdb1d89-a929-42c7-9e13-0edb88af4b24)

**Comment by rnchg** [comment]
网速有点慢，正在上传中……
________________________________
发件人: HumanMusic ***@***.***>
发送时间: 2024年7月8日 21:14
收件人: rnchg/Apt ***@***.***>
抄送: Subscribed ***@***.***>
主题: [rnchg/Apt] Google Drive 分享打开文件夹是空的。 (Discussion #5)


image.png (view on web)<https://github.com/rnchg/Apt/assets/46023154/0cdb1d89-a929-42c7-9e13-0edb88af4b24>

―
